payjoin-relay/src/db.rs

```diff
@@ -1,189 +1,355 @@
-use std::time::Duration;
+#![allow(unused_imports)]
+use std::{
+    fmt::Display,
+    pin::Pin,
+    sync::{Arc, Mutex},
+    thread,
+    time::Duration,
+};
 
 use anyhow::Result;
-use sqlx::postgres::{PgListener, PgPoolOptions};
-use sqlx::{PgPool, Pool, Postgres};
-use tracing::debug;
+use futures_util::StreamExt;
+use redis::{
+    aio::{self, AsyncStream},
+    AsyncCommands, Client, Commands, Connection, ControlFlow, PubSub, PubSubCommands, RedisError,
+};
 
-const RES_COLUMN: &str = "res";
-const REQ_COLUMN: &str = "req";
```

Comment:
changing names that don't affect operation might be a bit of a distraction for both you and the reviewers